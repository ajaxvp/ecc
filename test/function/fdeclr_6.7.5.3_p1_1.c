/* ISO: 6.7.5.3 (1); function declarator constraint: no function or array return type */

int (f(int x))[5];

int (f(int x))(void);
