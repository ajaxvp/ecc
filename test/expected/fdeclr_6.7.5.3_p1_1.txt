ecc: error: [3:1] functions may not specify array or function return types
ecc: error: [5:1] functions may not specify array or function return types
